projects/ccf-eui/src/themes/_dark.scss

```diff
@@ -70,7 +71,9 @@ $background: map-merge(mat.$dark-theme-background-palette, (
   mat-slider-primary: #ececec,
   info-button-hover: map-get($primary-colors, 850),
   checkbox-color: #636363,
-  button-background-hover: #2f3346
+  button-background-hover: #2f3346,
+  icon-background-hover: #2f3346,
```

Comment:
Need to verify these settings with @libby-maier.
Also needs improved naming!

Reply:
Let me know if you want to jump on a quick call @axdanbol & @edlu77!